dask/dataframe/tests/test_dataframe.py

```diff
@@ -762,8 +762,8 @@ def test_dropna():
     )
 
     # threshold
-    assert_eq(df.dropna(thresh=None), df.loc[[20, 40]])
-    assert_eq(ddf.dropna(thresh=None), df.dropna(thresh=None))
+    assert_eq(df.dropna(), df.loc[[20, 40]])
+    assert_eq(ddf.dropna(), df.dropna())
```

Comment:
Hrm, why did we drop `thresh=` here? It looks like it wasn't being used in conjunction with `how=` in this case (though I could be missing something)

Reply:
Could you also add a test that checks an error is raised when both `how` and `thresh` are specified?

Reply:
I think I was trying something and I forgot, and now I see that if they are there, we have an AssertionError for 
` assert_eq(df.dropna(thresh=None), df.loc[[20, 40]])` I have to take a deeper look. I'll push the code so it's visible.